english/marshmallow.md

```diff
@@ -274,3 +274,4 @@ Do you:
 [Eat the Marshmallow](eatmarshmallow/eatmarshmallow.md)
 
 [Escape into Willy Wonka Lego-World](lego-world/lego-world.md)
+[Eat breakfast](breakfast/breakfast.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.